learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "dc1e7b02-f022-48ca-a270-21f4b0c70835",
+    "_uuid": "0696f41450f17b2e0ebbbbb30a29053cde7de211"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Selecting and Data for Modeling\n",
+    "Your dataset had  too many variables to wrap your head around, or even to print out nicely.  How can you pare down this overwhelming amount of data to something you can understand?\n",
+    "\n",
+    "To show you the techniques, we'll start by picking a few variables using our intuition. Later tutorials will show you statistical techniques to  automatically prioritize variables.\n",
+    "\n",
+    "To choose variables/columns, we'll need to see a list of all columns in the dataset. That is done with the **columns** property of the DataFrame (the bottom line of code below).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "melbourne_file_path = '../input/melbourne-housing-snapshot/melb_data.csv'\n",
+    "melbourne_data = pd.read_csv(melbourne_file_path) \n",
+    "print(melbourne_data.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The Melbourne data has somemissing values (some houses for which some variables weren't recorded.)\n",
+    "# We'll learn to handle missing values in a later tutorial.  \n",
+    "# Your Iowa data doesn't have missing values in the predictors you use. \n",
+    "# So we will take the simplest option for now, and drop those houses from our data. \n",
+    "#Don't worry about this much for now, though the code is:\n",
+    "\n",
+    "# dropna drops missing values (think of na as \"not available\")\n",
+    "melbourne_data = melbourne_data.dropna(axis=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are many ways to select a subset of your data. The [Pandas Course](https://www.kaggle.com/learn/pandas) covers these in more depth. But we can get our data set up for modeling with just two of these approaches.\n",
+    "\n",
+    "1. Dot notation, which we use to select the \"prediction target\"\n",
+    "2. Selecting with a column list, which we use to select the \"predictors\"\n",
+    "\n",
+    "## Selecting The Prediction Target \n",
+    "You can pull out a variable with **dot-notation**.  This single column is stored in a **Series**, which is broadly like a DataFrame with only a single column of data.  \n",
+    "\n",
+    "We'll use the dot notation to select the column we want to predict, which is called the **prediction target**. By convention, the prediction target is called **y**. So the code we need to save the house prices in the Melbourne data is"
+   ]
```

Comment:
Big fan of bolding keywords like this. I'm an inveterate skimmer (and I bet lots of our readers are too)